diff --git a/ui/src/main/java/me/ledge/link/sdk/ui/presenters/link/WelcomePresenter.java b/ui/src/main/java/me/ledge/link/sdk/ui/presenters/link/WelcomePresenter.java
index b844056..1e35e11 100644
--- a/ui/src/main/java/me/ledge/link/sdk/ui/presenters/link/WelcomePresenter.java
+++ b/ui/src/main/java/me/ledge/link/sdk/ui/presenters/link/WelcomePresenter.java
@@ -101,7 +101,7 @@ public class WelcomePresenter
         GenericMessageConfigurationVo actionConfig = (GenericMessageConfigurationVo) welcomeScreenAction.configuration;
         mActivity.runOnUiThread(() -> {
             mActivity.setTitle(actionConfig.title);
-            mView.setCallToAction(actionConfig.callToAction.title);
+            mView.setCallToAction(actionConfig.callToAction.title.toUpperCase());
             setWelcomeText(actionConfig.content.value);
             if(actionConfig.image != null) {
                 mView.setImage(actionConfig.image);
